Python/MAD_LIBS_BOT/requirements.txt

```diff
@@ -0,0 +1,2 @@
+discord.py
```

Comment:
Create your `requirements.txt` as follows like [here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/Coderforces_Problem_Scrapper/requirements.txt). You can refer from [here](https://note.nkmk.me/en/python-pip-install-requirements/#:~:text=If%20you%20output%20pip%20freeze%20to%20a%20file%20with%20redirect%20%3E%2C%20you%20can%20use%20that%20file%20to%20install%20packages%20of%20the%20same%20version%20as%20the%20original%20environment%20in%20another%20environment)